Library/Homebrew/os/mac.rb

```diff
@@ -31,9 +31,17 @@ def full_version=(version)
       @version = nil
     end
 
+    def latest_version
+      Version.new "10.13"
```

Comment:
just checking that these versions won't ever have a patch number (10.13.1)?

Reply:
this gets compared to `version` not `full_version` so it should be ok I think

Reply:
yeah, it looks like version just has 10.minor

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/os/mac.rb#L19-L21

thanks!